.github/workflows/fuzzing.yml

```diff
@@ -10,9 +10,15 @@ on:
       - 'src/**'
       - 'test/fuzz/**'
       - 'test/static/corpus/**'
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
```

Comment:
(Can't comment the code outside the diff, that's about `branches` above.)

Should not we enable it on 2.8/1.10 and on tags as well? I asked it, because, I guess, the resulting rules should be very similar to ones created in PR #6630.

Reply:
^^^ in a separate commit:
```
Author: Sergey Bronnikov <sergeyb@tarantool.org>
Date:   Wed Jan 12 18:32:05 2022 +0300

    ci: run fuzzing on tags and release branches too
```

Reply:
> ^^^ in a separate commit:
> 
> ```
> Author: Sergey Bronnikov <sergeyb@tarantool.org>
> Date:   Wed Jan 12 18:32:05 2022 +0300
> 
>     ci: run fuzzing on tags and release branches too
> ```

Okay.